Common++/header/Logger.h

```diff
@@ -52,64 +55,64 @@ namespace pcpp
+		PacketLogModuleIcmpV6Layer,		///< IcmpV6Layer module (Packet++)
+		PacketLogModuleGreLayer,		///< GreLayer module (Packet++)
+		PacketLogModuleSSLLayer,		///< SSLLayer module (Packet++)
+		PacketLogModuleSllLayer,		///< SllLayer module (Packet++)
+		PacketLogModuleNflogLayer,		///< NflogLayer module (Packet++)
+		PacketLogModuleDhcpLayer,		///< DhcpLayer module (Packet++)
+		PacketLogModuleDhcpV6Layer,		///< DhcpV6Layer module (Packet++)
+		PacketLogModuleIgmpLayer,		///< IgmpLayer module (Packet++)
+		PacketLogModuleSipLayer,		///< SipLayer module (Packet++)
+		PacketLogModuleSdpLayer,		///< SdpLayer module (Packet++)
+		PacketLogModuleRadiusLayer,		///< RadiusLayer module (Packet++)
+		PacketLogModuleGtpLayer,		///< GtpLayer module (Packet++)
+		PacketLogModuleBgpLayer,		///< GtpLayer module (Packet++)
+		PacketLogModuleSSHLayer,		///< SSHLayer module (Packet++)
+		PacketLogModuleVrrpLayer,		///< Vrrp Record module (Packet++)
+		PacketLogModuleTcpReassembly,	///< TcpReassembly module (Packet++)
+		PacketLogModuleIPReassembly,	///< IPReassembly module (Packet++)
+		PacketLogModuleIPSecLayer,		///< IPSecLayers module (Packet++)
+		PacketLogModuleNtpLayer,		///< NtpLayer module (Packet++)
+		PacketLogModuleTelnetLayer,		///< TelnetLayer module (Packet++)
+		PacketLogModuleStpLayer,		///< StpLayer module (Packet++)
+		PacketLogModuleLLCLayer,		///< LLCLayer module (Packet++)
+		PacketLogModuleNdpLayer,		///< NdpLayer module (Packet++)
+		PacketLogModuleFtpLayer,		///< FtpLayer module (Packet++)
+		PacketLogModuleSomeIpLayer,		///< SomeIpLayer module (Packet++)
+		PacketLogModuleSomeIpSdLayer,	///< SomeIpSdLayer module (Packet++)
+		PacketLogModuleWakeOnLanLayer,	///< WakeOnLanLayer module (Packet++)
+		PacketLogModuleSmtpLayer,		///< SmtpLayer module (Packet++)
 		PcapLogModuleWinPcapLiveDevice, ///< WinPcapLiveDevice module (Pcap++)
-		PcapLogModuleRemoteDevice, ///< WinPcapRemoteDevice module (Pcap++)
-		PcapLogModuleLiveDevice, ///< PcapLiveDevice module (Pcap++)
-		PcapLogModuleFileDevice, ///< FileDevice module (Pcap++)
-		PcapLogModulePfRingDevice, ///< PfRingDevice module (Pcap++)
-		PcapLogModuleMBufRawPacket, ///< MBufRawPacket module (Pcap++)
-		PcapLogModuleDpdkDevice, ///< DpdkDevice module (Pcap++)
-		PcapLogModuleKniDevice, ///< KniDevice module (Pcap++)
-		PcapLogModuleXdpDevice, ///< XdpDevice module (Pcap++)
-		NetworkUtils, ///< NetworkUtils module (Pcap++)
+		PcapLogModuleRemoteDevice,		///< WinPcapRemoteDevice module (Pcap++)
+		PcapLogModuleLiveDevice,		///< PcapLiveDevice module (Pcap++)
+		PcapLogModuleFileDevice,		///< FileDevice module (Pcap++)
+		PcapLogModulePfRingDevice,		///< PfRingDevice module (Pcap++)
+		PcapLogModuleMBufRawPacket,		///< MBufRawPacket module (Pcap++)
+		PcapLogModuleDpdkDevice,		///< DpdkDevice module (Pcap++)
+		PcapLogModuleKniDevice,			///< KniDevice module (Pcap++)
+		PcapLogModuleXdpDevice,			///< XdpDevice module (Pcap++)
+		NetworkUtils,					///< NetworkUtils module (Pcap++)
```

Comment:
Also here - I'm not sure why clang-format decided to use tabs instead of spaces. It doesn't look as it should:

![image](https://github.com/seladb/PcapPlusPlus/assets/9059541/401a955b-2170-455a-a445-23cd4b21dbe9)

Reply:
fixed.